documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -0,0 +1,1047 @@
+apiVersion: forklift.konveyor.io/v1beta1
+kind: Hook
+metadata:
+  name: <hook>
+  namespace: <namespace>
+spec:
+  image: quay.io/konveyor/hook-runner
+  playbook: |
+    LS0tCi0gbmFtZTogTWFpbgogIGhvc3RzOiBsb2NhbGhvc3QKICB0YXNrczoKICAtIG5hbWU6IExv
+    YWQgUGxhbgogICAgaW5jbHVkZV92YXJzOgogICAgICBmaWxlOiAiL3RtcC9ob29rL3BsYW4ueW1s
+    IgogICAgICBuYW1lOiBwbGFuCiAgLSBuYW1lOiBMb2FkIFdvcmtsb2FkCiAgICBpbmNsdWRlX3Zh
+    cnM6CiAgICAgIGZpbGU6ICIvdG1wL2hvb2svd29ya2xvYWQueW1sIgogICAgICBuYW1lOiB3b3Jr
+    bG9hZAoK
+EOF
+----
++
+where:
++
+`playbook` refers to an optional Base64-encoded Ansible playbook. If you specify a playbook, the `image` must be `hook-runner`.
++
+[NOTE]
+====
+You can use the default `hook-runner` image or specify a custom image. If you specify a custom image, you do not have to specify a playbook.
+====
++
+
+. Create a `Plan` manifest for the migration:
++
+ifdef::vmware[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: Plan
+metadata:
+  name: <plan> <1>
+  namespace: <namespace>
+spec:
+  warm: true <2>
+  provider:
+    source:
+      name: <source_provider>
+      namespace: <namespace>
+    destination:
+      name: <destination_provider>
+      namespace: <namespace>
+  map: <3>
+    network: <4>
+      name: <network_map> <5>
+      namespace: <namespace>
+    storage: <6>
+      name: <storage_map> <7>
+      namespace: <namespace>
+  targetNamespace: <target_namespace>
+  vms: <8>
+    - id: <source_vm> <9>
+    - name: <source_vm>
+      hooks: <10>
+        - hook:
+            namespace: <namespace>
+            name: <hook> <11>
+          step: <step> <12>
+EOF
+----
+<1> Specify the name of the `Plan` CR.
+<2> Specify whether the migration is warm or cold. If you specify a warm migration without specifying a value for the `cutover` parameter in the `Migration` manifest, only the precopy stage will run.
+<3> Specify only one network map and one storage map per plan.
+<4> Specify a network mapping even if the VMs to be migrated are not assigned to a network. The mapping can be empty in this case.
+<5> Specify the name of the `NetworkMap` CR.
+<6> Specify a storage mapping even if the VMs to be migrated are not assigned with disk images. The mapping can be empty in this case.
+<7> Specify the name of the `StorageMap` CR.
+<8> You can use either the `id` _or_ the `name` parameter to specify the source VMs. +
+<9> Specify the VMware VM MOR..
+<10> Optional: You can specify up to two hooks for a VM. Each hook must run during a separate migration step.
+<11> Specify the name of the `Hook` CR.
+<12> Allowed values are `PreHook`, before the migration plan starts, or `PostHook`, after the migration is complete.
+endif::[]
+
+ifdef::rhv[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: Plan
+metadata:
+  name: <plan> <1>
+  namespace: <namespace>
+spec:
```

Comment:
we're going to have another field here `preserveClusterCpuModel`, see explanation [here](https://github.com/kubev2v/forklift/pull/688/files#diff-09e2c8040fb3d68c4ad0656289202f8f305f29f8973eef899ca8d5efc947cba4R47)